### PR TITLE
release: 0.8.4

### PR DESCRIPTION
**Pull request by audrium**
Reverts changes done in https://github.com/reanahub/reana-commons/pull/333

addresses https://github.com/reanahub/reana-workflow-engine-yadage/issues/222

### CODE REVIEW

**Comment by tiborsimko** [comment]
A new `r-commons` release could include the patch from https://github.com/reanahub/reana-commons/pull/269 that is starting to build OpenShift compliance.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.8.4 (2022-02-08)
+--------------------------
+
+- Changes installation to revert ``Yadage`` dependency versions.
```
Comment (mvidalgarcia) [inline CHANGES.rst:7]: Should we add a new entry for https://github.com/reanahub/reana-commons/commit/4f15ed4857bb57e79b842d0381a073cc05720ce0?

**Comment by mvidalgarcia** [comment]
Perhaps we can include [this issue reference](https://github.com/reanahub/reana-workflow-engine-yadage/issues/222) in the revert commit.

**Review by mvidalgarcia — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def get_snakemake_pkg(extras=""):
     "docs": ["Sphinx>=1.4.4", "sphinx-rtd-theme>=0.1.9",],
     "tests": tests_require,
     "kubernetes": ["kubernetes>=11.0.0,<12.0.0",],
-    "yadage": ["adage~=0.10.3", "yadage~=0.20.2", "yadage-schemas~=0.10.6"],
+    "yadage": ["adage~=0.10.1", "yadage~=0.20.1", "yadage-schemas~=0.10.6"],
```
Comment (matthewfeickert) [inline setup.py:35]: There was no need to revert the `adage` bit here.

**Pull request merged**